src/app/components/client/Button.module.scss

```diff
@@ -216,6 +216,10 @@
     line-height: 1.5;
     padding: tokens.$spacing-sm tokens.$spacing-md;
 
+    &.link {
+      padding: 0;
+    }
```

Comment:
Technically, this is not needed as the combination of props `variant="link"` and `small` is not supposed to exist.